### PR TITLE
Prepare 2.0.0 release

### PR DESCRIPTION
**Pull request by BClark09**
Test Outputs: https://bintray.com/bclark/apt-repo2/openhab2/2.0.0

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

**Comment by BClark09** [comment]
### Systemd 
 - New install - Working.
 - Upgrade from RC1 - Working.

### sysVinit

 - New install - Working.
 - Upgrade from RC1 - Requires reboot of OS, otherwise working.

**Comment by ThomDietrich** [comment]
How about "Downgrade from newer snapshot" ? Could be interesting in the next few days...

**Comment by BClark09** [comment]
To change to any version, add the stable repo to sources, then:

```
sudo apt-get update
sudo apt-get install openhab2=2.0.0-1
```

I'll add a generic "switch versions".

File: build.gradle [hunk 1]
```diff
@@ -60,6 +60,33 @@ def distributions = [
                          "version": "2.0.0~b5",
                          "release": '1'
                         ],
+                        ["dist": "openhab2-release",
```
Comment (theoweiss) [inline build.gradle]: I would prefer to call it openhab2-2.0.0 or openhab-2_0_0/openhab-200 if this does not work. So we are prepared for a 2.1.0 release.

File: build.gradle [hunk 2]
```diff
@@ -274,6 +303,8 @@ task buildBeta5(dependsOn: tasks.findAll { t -> t.name.endsWith("-b5")})
 
 task buildRC1(dependsOn: tasks.findAll { t -> t.name.endsWith("-rc1")})
 
+task buildRelease(dependsOn: tasks.findAll { t -> t.name.endsWith("-release")})
```
Comment (theoweiss) [inline build.gradle]: Would prefer naming it `buildLatestRelease(dependsOn: tasks.findAll { t -> t.name.endsWith("-2.0.0")})`. We could move this target to 2.1.0 easily  to always point to the latest release.

File: build.gradle [hunk 3]
```diff
@@ -60,6 +60,33 @@ def distributions = [
...
+                        ["dist": "openhab2-release",
+                         "description": "Linux installation package for openHAB 2.",
+                         "debDist": "stable",
+                         "packageName": "openhab2",
+                         "url": "https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab%2F2.0.0%2Fopenhab-2.0.0.tar.gz",
+                         "path": buildDir.getAbsolutePath() + '/' + 'openhab-2.0.0.tar.gz',
+                         "version": "2.0.0",
+                         "release": '1'
+                        ],
+                        ["dist": "openhab2-addons-release",
```
Comment (theoweiss) [inline build.gradle]: Also 2.0.0 here.

File: build.gradle [hunk 4]
```diff
@@ -60,6 +60,33 @@ def distributions = [
...
+                         "description": "Linux installation package for openHAB 2 addons.",
+                         "debDist": "stable",
+                         "packageName": "openhab2-addons",
+                         "url": "https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab-addons%2F2.0.0%2Fopenhab-addons-2.0.0.kar",
+                         "path": buildDir.getAbsolutePath() + '/' + 'openhab-addons-2.0.0.kar',
+                         "version": "2.0.0",
+                         "release": '1'
+                        ],
+                        ["dist": "openhab2-addons-legacy-release",
```
Comment (theoweiss) [inline build.gradle]: Also 2.0.0 here.

**Comment by theoweiss** [comment]
Lgtm. Are we ready to merge?

**Comment by BClark09** [comment]
Done, thanks @theoweiss, "2.0.0" builds and installs fine.

**Comment by theoweiss** [comment]
Ok. I'll merge this PR and check if I can find a solution for the sysVinit issue.

**Comment by BClark09** [comment]
I think we're good to merge. To do _after_ the nightly build has been triggered, change snapshot 2.0.0 to 2.1.0

**Pull request merged**